aws/resource_aws_security_group_rule_test.go

```diff
@@ -154,6 +154,32 @@ func TestAccAWSSecurityGroupRule_Ingress_VPC(t *testing.T) {
 	})
 }
 
+func TestAccAWSSecurityGroupRule_Ingress_Source_With_Account_Id(t *testing.T) {
+	var group ec2.SecurityGroup
+
+	rInt := acctest.RandInt()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSSecurityGroupRuleDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSSecurityGroupRule_Ingress_Source_with_AccountId(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSSecurityGroupRuleExists("aws_security_group.web", &group),
+				),
+			},
+			// Ensure plan shows no difference after state is refreshed
+			{
+				Config:             testAccAWSSecurityGroupRule_Ingress_Source_with_AccountId(rInt),
+				PlanOnly:           true,
+				ExpectNonEmptyPlan: false,
+			},
```

Comment:
This is extraneous as the acceptance testing framework automatically checks a refresh plan for differences after applying a `Config` 👍 

```suggestion
```